src/wallet/wallet.cpp

```diff
@@ -2940,7 +2940,7 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CTransac
                 const OutputType change_type = TransactionChangeType(coin_control.m_change_type ? *coin_control.m_change_type : m_default_change_type, vecSend);
                 // One change script per output asset.
                 size_t index = 0;
-                for (const std::pair<CAsset, CAmount>& value : mapValue) {
+                for (const std::pair<const CAsset, CAmount>& value : mapValue) {
```

Comment:
compiler complaint squashing could be its own commit

Reply:
I think this must have gone away in the rebase

Reply:
ah yeah